storage/buckets/create_bucket_dual_region.go

```diff
@@ -25,10 +25,11 @@ import (
 )
 
 // createBucketDualRegion creates a new dual-region bucket in the project in the
-// provided locations.
-func createBucketDualRegion(w io.Writer, projectID, bucketName, region1, region2 string) error {
+// provided location and regions.
+func createBucketDualRegion(w io.Writer, projectID, bucketName, location, region1, region2 string) error {
```

Comment:
We're supposed to limit function the args passed into samples to the minimum required for testing. Therefore, let's define location, region1 and region2 all in the func body rather than using args. See https://googlecloudplatform.github.io/samples-style-guide/#minimal-arguments

Reply:
Thanks for linking! Updated to define vars inside the func body.